packages/input-amount/src/LionInputAmount.js

```diff
@@ -90,6 +91,20 @@ export class LionInputAmount extends LocalizeMixin(LionInput) {
     }
   }
 
+  /**
+   * Converts `.modelValue` to `.serializedValue`, which needs to be the same amount of decimals as the formattedValue
+   * @param {number} modelValue
+   * @override FormatMixin
+   */
+  serializer(modelValue) {
+    if (modelValue === undefined) {
+      return '';
+    }
+    const formatted = formatAmount(modelValue, this.formatOptions);
```

Comment:
By doing it like this, extension that override the formatter may get out of sync... but also see remarks below. We might need to solve it on modelValue level and make the serializedValue benefit from that